test-lit/lit.cfg.py

```diff
@@ -26,17 +26,22 @@
 
 
 # suffixes: A list of file extensions to treat as test files.
-config.suffixes = ['.f', '.F', '.ff','.FOR', '.for', '.f77', '.f90', '.F90',
+config.suffixes = ['.f', '.F', '.ff', '.FOR', '.for', '.f77', '.f90', '.F90',
                    '.ff90', '.f95', '.F95', '.ff95', '.fpp', '.FPP', '.cuf',
                    '.CUF', '.f18', '.F18']
 
 # test_source_root: The root path where tests are located.
 config.test_source_root = os.path.dirname(__file__)
 
-
 # test_exec_root: The root path where tests should be run.
 config.test_exec_root = os.path.join(config.flang_obj_root, 'test-lit')
 
+config.old_test_source_root = os.path.join(
+    config.test_source_root, "..", "test")
```

Comment:
Alternative to this approach is to copy over the old test scripts to the new test directory. Then you can use config.test_source_root.

However, would a neater way to be to use lit's %S variable which expands to the test directory. Your script would insert %S/test_errors.sh instead of %test_error. This could be expanded to all the testing tools without need to plumb them all through lit. I could be wrong, but I think the advantages of the lit tool expansion don't apply to a script (rather than executable) that is sitting next to the test inputs (rather than in some other directory).

Reply:
Might be good to isolate the scripts in their own directory separate from the tests.